Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1347,6 +1355,31 @@ - (void)processNextQueueItem {
     }
 }
 
+- (void)getBranchUniversalLinkUrlForThirdPartyUrl {
```

Comment:
Sounds good. I should probably add code to use the webview's user agent string as well